scenarios/aws/ec2/vm_run.go

```diff
@@ -41,6 +41,10 @@ func VMRun(ctx *pulumi.Context) error {
 				fakeIntakeOptions = append(fakeIntakeOptions, fakeintake.WithLoadBalancer())
 			}
 
+			if env.AgentUseDualShipping() {
+				fakeIntakeOptions = append(fakeIntakeOptions, fakeintake.WithoutDDDevForwarding())
+			}
```

Comment:
We can remove that one as well, we cannot configure dual shipping for the agent on VM